public/js/showDetailBook.js

```diff
@@ -0,0 +1,31 @@
+url = window.location.pathname;
+var id = url.substring(url.lastIndexOf('/') + 1);
+$.ajax({
+    url: "/api/books/" + id,
+    type: "get",
+    success: function( data ) {
+        for(var i = 0; i <= 3; i++){
+            if(typeof data.image_books[i] === 'undefined') {
```

Comment:
Nếu book đã không có image thì đâu cần phải dùng for loop?
Đoạn này cứ nên để image default ngoài html. Book có image thì mới thay đổi.